src/components/city.js

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink, useLocation } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { fetchCities } from '../Redux/Reducers/Cities';
+import '../App.css';
+
+const Cities = () => {
+  const dispatch = useDispatch();
+  const location = useLocation();
+  const countryInfo = location.state.info;
+  useEffect(() => {
+    dispatch(fetchCities(countryInfo.name));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dispatch]);
```

Comment:
I noticed you disabled eslint in this file, it is against microverse policy to disable linters for linters help us write clean code. it would be nice if you enabled eslint.